src/main/java/org/mockito/plugins/MockitoLogger.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2007 Mockito contributors
  * This program is made available under the terms of the MIT License.
  */
-package org.mockito.internal.util;
+package org.mockito.plugins;
 
 public interface MockitoLogger {
```

Comment:
We are going to need some additional documentation here. https://static.javadoc.io/org.mockito/mockito-core/2.23.4/org/mockito/plugins/MockMaker.html is probably a good starting point of what we can add as documentation.